parser/build.rs

```diff
@@ -1,13 +1,10 @@
 use std::fmt::Write as _;
 use std::fs::File;
-use std::io::{BufRead, BufReader, BufWriter, Write};
+use std::io::{BufRead, BufReader};
 use std::path::{Path, PathBuf};
 use tiny_keccak::{Hasher, Sha3};
 
 fn main() -> anyhow::Result<()> {
```

Comment:
cargo for some reason loves to recompile the parser on every opportunity (i don't want to dig even deeper into cargo's cache invalidation)

Reply:
I never noticed this. Investigating why could be a worthwhile exercise but I'm not planning to address it in this PR